queries/angular/highlights.scm

```diff
@@ -21,47 +42,82 @@
   function: ((identifier) @function.builtin
     (#eq? @function.builtin "$any")))
 
+(pair
+  key: ((identifier) @variable.builtin
+  (#eq? @variable.builtin "$implicit")))
+
+(control_keyword) @type.control
+(special_keyword) @type
+
+((identifier) @boolean
+  (#any-of? @boolean "true" "false"))
+((identifier) @variable.builtin
+  (#any-of? @variable.builtin "this" "\$event" "null"))
+
 [
-"let"
-"as"
-] @keyword
+ (ternary_operator)
+ (conditional_operator)
+] @conditional.ternary
 
+;
+; (if_statement "\@" @type)
 [
-"("
-")"
-"["
-"]"
-"{"
-"}"
+  "("
+  ")"
+  "["
+  "]"
+  "{"
+  "}"
+  "{{"
+  "}}"
+  (if_end_expression)
+  (for_end_expression)
+  (switch_end_expression)
+  (case_end_expression)
+  (default_end_expression)
+  (defer_end_expression)
 ] @punctuation.bracket
 
+(start_tag
+  [ "<" ">" "/>"] @tag.delimiter
+)
+(self_closing_tag
+  [ "<" ">" "/>"] @tag.delimiter
+)
+(end_tag
+  [ "</" ">" ] @tag.delimiter
+)
+(erroneous_end_tag
+  [ "</" ">" ] @tag.delimiter
+)
```

Comment:
Are these not highlighted via `html_tags` already?

Reply:
You are right.